web/src/pages/api/public/ingestion.ts

```diff
@@ -244,6 +245,41 @@ export default async function handler(
       });
     }
 
+    // Send each event individually to IngestionQueue for new processing
```

Comment:
i think we should add here another env to be able to determine whether we want to send these events or not.

Reply:
I think we never want to not upload anymore to have full history in S3. But we might want to turn off the processing.

Reply:
@maxdeichmann Shall we just check whether the CLICKHOUSE_URL is set? That was the "gate" in the old flow.

Reply:
I added LANGFUSE_ASYNC_CLICKHOUSE_INGESTION_PROCESSING for gating it.